src/tdlib/chat_action_list.rs

```diff
@@ -116,10 +116,9 @@ impl ChatActionList {
     pub(crate) fn last(&self) -> Option<ChatAction> {
         self.imp()
             .list
-            .borrow()
-            .last()
-            .map(|(_, action)| action)
-            .cloned()
+            .try_borrow()
+            .ok()
+            .and_then(|l| l.last().map(|(_, action)| action.clone()))
```

Comment:
This doesn't seem to be correct.

Reply:
Basically, I've changed it because it was borrowed mutable multiple times and I didn't know how to fix this. I realized my workaround causes new problems, but left it for now since this pr is not ready to merge anyway.